gdscript-customization.el

```diff
@@ -138,5 +148,21 @@ directory path containing the file `index.html'."
   :type 'string
   :group 'gdscript)
 
+(defcustom gdscript-docs-online-api-url "https://docs.godotengine.org/en/stable/classes/index.html?#godot-api"

```

Comment:
Do we need these variables to be customizable?

The only case I can think of is browsing docs for specific versions of Godot, like "latest" or 3.1. In which case we shouldn't expose the full string templates as a public variable, but rather an enum of available versions.

I'd generally keep values like `https://docs.godotengine.org/en/stable/classes/class_%s.html#%s` private as the functions rely on the template to work.